package_build_list_host_linux.json

```diff
@@ -5,8 +5,8 @@
     "comment4" : "Note:  Build from source occurs before build_from_folder",
     "build_from_source": {
         "assimp-5.2.5-rev1-linux":  "Scripts/extras/pull_and_build_from_git.py ../../package-system/assimp --platform-name Linux --package-root ../../package-system --clean",
-        "AWSGameLiftServerSDK-5.0.0-rev3-linux-aarch64": "Scripts/extras/pull_and_build_from_git.py ../../package-system/AWSGameLiftServerSDK --platform-name Linux-aarch64 --clean",
-        "AWSGameLiftServerSDK-5.0.0-rev3-linux": "Scripts/extras/pull_and_build_from_git.py ../../package-system/AWSGameLiftServerSDK --platform-name Linux --clean",
+        "AWSGameLiftServerSDK-5.1.1-rev1-linux-aarch64": "Scripts/extras/pull_and_build_from_git.py ../../package-system/AWSGameLiftServerSDK --platform-name Linux-aarch64 --clean",
```

Comment:
Should this be in the Linux json? It's already in the Linux aarch64 one.

Reply:
Removed!

Building ARM on x86 is supported, but only if Docker+QEMU is setup. The QEMU step is currently being skipped.
https://github.com/o3de/3p-package-source/issues/234

Reply:
Reverting. Aarch64 needs to be included in the linux x86 build list because GitHub workflow doesn't support native ARM (so we build aarch on an x86 using docker)